src/index.js

```diff
@@ -90,7 +90,9 @@ class PubSubRoom extends EventEmitter {
     })
 
     this.once('stop', () => {
-      this._ipfs.pubsub.unsubscribe(this._topic, listener)
+      if (this._ipfs.state.stopped()) {
```

Comment:
isn't the logic reversed? Shouldn't we unsubscribe is the IPFS node hasn't stopped? This would prevent unsubscribing normally, no?

Reply:
You're absolutely right! 😄 Then the question is, why does this fix the race? Need to take another look.